src/Microsoft.ML.Tokenizers/Model/BpeTrainer.cs

```diff
@@ -1,534 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Runtime.CompilerServices;
-using System.Text;
-
-namespace Microsoft.ML.Tokenizers
-{
-    /// <summary>
-    /// The Bpe trainer responsible to train the Bpe model.
-    /// </summary>
-    public sealed class BpeTrainer : Trainer
```

Comment:
FWIW, I don't have a problem with having BPE training support if we believe it's valuable and something folks are actually using. I just question it being part of the abstraction for tokenizers, and therefore needing an abstraction for trainers (e.g. do folks need to train models without knowing what kind of models they're using?) We can add it back later if needed.

Reply:
Yes, we can consider adding it back if we see any request for that. So far I am not aware of anyone using it. Usually users picks the data from already baked models.